src/demo-app/accessibility/button/button-accessibility.html

```diff
@@ -0,0 +1,23 @@
+<div class="demo-button">
+  <section>
```

Comment:
Each section should have a heading `<h2>` and probably a short paragraph `<p>` describing what the demo highlights. Something like
```html
<h2>Buttons</h2>
<p>Examples of all of Angular Material's button variants</p>
<button md-button>Activate basic button</button>
<button md-raised-button>Activate button with raised style</button>
```